profiles/common/modules/features/multisite_custom_error/multisite_custom_error.info

```diff
@@ -8,3 +8,4 @@ dependencies[] = features
 dependencies[] = customerror
 features[features_api][] = api:2
 files[] = tests/multisite_custom_error.extra.test
+configure = admin/config/system/customerror
```

Comment:
This is the config page of ``customerror`` module, it must be remove from ``multisite_custom_error`` feature.